geanydoc/src/geanydoc.c

```diff
@@ -80,24 +78,17 @@ word_check_right(gchar c)
 static gchar *
 current_word(void)
 {
-	gchar *txt;
 	GeanyDocument *doc;
 
 	gint pos;
 	gint cstart, cend;
 	gchar c;
-	gint text_len;
 
 	doc = document_get_current();
 	g_return_val_if_fail(doc != NULL && doc->file_name != NULL, NULL);
 
-	text_len = sci_get_selected_text_length(doc->editor->sci);
-	if (text_len > 1)
```

Comment:
Technically this checks for 2-character selections and longer but looks more like a bug than a feature.